tests/php/sync/test_class.jetpack-sync-callables.php

```diff
@@ -1001,6 +1002,7 @@ function mock_authenticated_xml_rpc_cleanup( $user_id ) {
 		unset( $_SERVER['REQUEST_METHOD'] );
 		$jetpack = Jetpack::init();
 		$jetpack->reset_saved_auth_state();
```

Comment:
Should we use the `reset_saved_auth_state` from the connection manager here too?

Reply:
No, this will also cleanup the rest auth status, so let's leave it as-is for now.